.changeset/khaki-hairs-vanish.md

```diff
@@ -0,0 +1,8 @@
+---
+"fluid-framework": patch
+"@fluidframework/map": patch
+---
+
+Map API documentation updates
+
+Remaining top-level APIs within `@fluidframework/map` have had docs updated to reflect their sealed nature, i.e. they are not to be implemented externally to Fluid Framework.
```

Comment:
Note for @microsoft/fluid-cr-docs follow-up - does not block this PR:

- Other "metadata" that's interesting in this case: The change is type-only. No runtime impact.
- Needs to help customers understand: Is this change relevant to me? How does it affect me? and What do I need to do next?
- "In the unlikely event that you have external implementations of these interfaces, note that these are unsupported." (?)

Reply:
This isn't even a type-only change. This is a documentation-only change. No types were harmed in the production of this change.